spotlight-client/src/VizControls/VizControls.tsx

```diff
@@ -91,6 +95,11 @@ const VizControls = ({
             // eslint-disable-next-line react/no-array-index-key
             filter && <FilterWrapper key={index}>{filter}</FilterWrapper>
         )}
+        {smallData && (
+          <TooltipTrigger maxWidth={232} contents={tenant?.smallDataDisclaimer}>
+            <Icon kind={IconSVG.Info} width={16} height={16} />
+          </TooltipTrigger>
+        )}
```

Comment:
Hey, @macfarlandian! Is there any way to style the Tooltip when using TooltipTrigger from design-system?

Reply:
not the tooltip itself, no, only the contents ... it is a known issue that unfortunately the design of tooltip contents in Spotlight does not really work with the tooltip from the design system. Hasn't been much of an issue in Spotlight thus far because there were no tooltips outside of the charts, but if we are adding that use case here, maybe it's better to add this feature to the design system than to reimplement the component. I know @terryttsai has looked into this recently so curious what he thinks

Reply:
cc @terryttsai

Reply:
I remember that the public dashboard's tooltip is designed differently than the one in design-system. Originally I had a story to move our tooltip to the one in design-system but ran into the issue in that I couldn't style it to match our design. Might make sure to modify the design-system's tooltip so that it can be styled properly. Either that or continue to use our own tooltip in this codebase...

Reply:
@terryttsai @macfarlandian I think it would be nice if we can properly style the Tooltip from the design-system. Could you create an Issue for that?

Reply:
done! https://github.com/Recidiviz/web-libraries/issues/105